gef-ui/server/utils/facility-helpers.js

```diff
@@ -96,18 +96,36 @@ const areUnissuedFacilitiesPresent = (application) => {
   return hasUnissuedFacilities;
 };
 
+// adds 3 months to date and returns formatted string deadline date
+const formatIssueDeadlineDate = (date) => {
+  const deadlineDate = add(new Date(date), { months: 3 });
+
+  return format(deadlineDate, 'dd MMM yyyy');
+};
+
 /*
    This function sets the deadline to display for unissued
    facilities on the unissued facilities table 3 months
-   from date of submission
+   from date of submission or MIN submission or now based on submission type
 */
-const facilityIssueDeadline = (submissionDate) => {
-  if (submissionDate) {
-  // converts to timestamp from epoch - '+' to convert from str to int
-    const date = new Date(parseInt(submissionDate, 10));
-    const deadlineDate = add(new Date(date), { months: 3 });
+const facilityIssueDeadline = (application) => {
+  // if MIA, then deadline should be 3 months from today
+  if (application.submissionType === CONSTANTS.DEAL_SUBMISSION_TYPE.MIA) {
+    return formatIssueDeadlineDate(new Date());
```

Comment:
Argument should be empty, so `new Date()` is then further now as passed as a value to `Date` constructor on LOC:101.
Suggestion: Amend the `formatIssueDealineDate` function to accept null or void argument, if so then use `new Date()` else use `new Date(date)`.

Reply:
done.  Removed many new dates as not required so many times.  Also used || to check if exists or not and either uses date or new Date()